test/TEST-04-JOURNAL/test.sh

```diff
@@ -76,9 +76,10 @@ EOF
 }
 
 test_cleanup() {
-    umount $TESTDIR/root 2>/dev/null
-    [[ $LOOPDEV ]] && losetup -d $LOOPDEV
+    umount $TESTDIR/root 2>/dev/null || true
+    [ -z "$LOOPDEV" ] || losetup -d $LOOPDEV || true
     return 0
 }
 
+
```

Comment:
Sorry, there are a few instances of this stray empty line here. But after this I'm going to factorize the scripts anyway, will clean these up then.